gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,3 +3,4 @@ distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionSha256Sum=e5444a57cda4a95f90b0c9446a9e1b47d3d7f69057765bfb54bd4f482542d548
```

Comment:
And another trailing newline issue.

Reply:
what is this problem really about?

Reply:
I just quote from Stackoverflow:

> It indicates that you do not have a newline (usually '\n', aka CR or CRLF) at the end of file.
>
> That is, simply speaking, the last byte (or bytes if you're on Windows) in the file is not a newline.
>
> The message is displayed because otherwise there is no way to tell the difference between a file where there is a newline at the end and one where is not. Diff has to output a newline anyway, or the result would be harder to read or process automatically.
>
> Note that it is a good style to always put the newline as a last character if it is allowed by the file format. Furthermore, for example, for C and C++ header files it is required by the language standard.
>
> from https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log